test/e2e/tsdx-build-default.test.ts

```diff
@@ -105,6 +105,15 @@ describe('tsdx build :: zero-config defaults', () => {
     shell.mv('package-og.json', 'package.json');
   });
 
+  it('should replace lodash/fp correctly', () => {
+    const output = execWithCache('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+
+    // ensures correct lodash/fp import was used
```

Comment:
```suggestion
```

the other tests don't have comments in here and I don't think this comment is necessary; the `it` block describes the check already